src/components/Trade/hooks/useSwapper/utils.ts

```diff
@@ -179,6 +180,14 @@ export const getDefaultAssetIdPairByChainId = (
             buyAssetId: 'eip155:56/bep20:0xe9e7cea3dedca5984780bafc599bd69add087d56',
           }
         : ethFoxPair
+    case KnownChainIds.PolygonMainnet:
+      return featureFlags.ZrxPolygonSwap
+        ? {
+            sellAssetId: polygonAssetId,
+            // USDC on Polygon
+            buyAssetId: 'eip155:137/erc20:0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174',
```

Comment:
Was going to say we could use FOX on Polygon here but looks like it was just bridged